modules/op-building-images-for-tekton-hub.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="building-images-for-tekton-hub_{context}"]
+= Building images for {tekton-hub}
+
+[role="_abstract"]
+Before deploying {tekton-hub}, you can build and push images to a registry.
```

Comment:
why are including manual steps in docs?
building images and deploying

we are integrating hub with operator to avoid customers need to build images and deploy

Reply:
docs should consist
after installing openshift pipelines operator, how to install hub on cluster (which is using TektonHub CR) and how to provide the required configuration for hub

Reply:
+1 on what @sm43 commented, we shouldn't include manual steps to deploy hub in docs

Reply:
Stuff from these [docs](https://github.com/tektoncd/operator/blob/main/docs/TektonHub.md) would be enough